index.html

```diff
@@ -0,0 +1,130 @@
+                    <a class="CC" href="#">CC Campaign</a>
+                </aside>
+            </section>
+            <a href="#" class="logo dk">BLK-NG</a>
+            <article class="main-nav-link dk">
+                <a id="about" href="./pages/about.html">About</a>
+                <a id="program" href="#">Program</a>
+                <a id="join" href="#">Join</a>
+                <a id= "sponsor" href="#">Sponsor</a>
+                <a id="news" href="#">News</a>
+                <a class="CC" href="#">CC Campaign</a>
+            </article>
+        </nav>
+        <section class="headline-container">
+            <article class="headline">
+                <h2>"Hello! Sharing world"</h2>
+                <h2>CREATIVE COMMONS<br>GLOBAL SUBMMIT<br>2023</h2>
+                <p class="intro-txt">
+                    A joyfull celebration believing in the value of openness and sharing creating 
+                    a positive change with people from over 80 countries is atking palce in Octobar, in Lagos
+                </p>
+                <h2 class="time">2023.10.15(THU) ~ 16(FRI)</h2>
+                <p>@ National Museum of Nigeria, Art Center Lagos and more</p>
+            </article>
+        </section>
+    </header>
+    <section id="main-program">
+        <h2 class="program-title wht">Main Program</h2>
+        <section id="cards">
+            <article class="card">
+                <img src="./assets/icons/home_iot_device_FILL0_wght400_GRAD0_opsz48.svg" alt="home_iot_device">
+                <h2>Home IoT</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+            <article class="card">
+                <img src="./assets/icons/smart_outlet_FILL0_wght400_GRAD0_opsz48.svg" alt="smart_outlet">
+                <h2>Smart Outlet</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+            <article class="card">
+                <img src="./assets/icons/assistant_on_hub_FILL0_wght400_GRAD0_opsz48.svg" alt="home_assistance">
+                <h2>Home Assistance</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+            <article class="card">
+                <img src="./assets/icons/rocket_launch_FILL0_wght400_GRAD0_opsz48.svg" alt="rocket_science">
+                <h2>Rocket Science</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+        </section>
+        <h2 class="Program-foot dk">SEE THE WHOLE PROGRAM</h2>
+        <h2 class="Program-foot mb">Join CC Global Summit 2023</h2>
```

Comment:
- [ ] 👋 Thank you for your efforts in the project. Upon reviewing your code, I noticed that you are using h2 tags for elements that should ideally be buttons or links as per the design. To follow best practices and improve the accessibility of your website, I recommend updating those elements to be buttons or links instead of h2 tags. By using the appropriate tags, such as buttons or links, you will ensure that users can easily identify interactive elements on your website, leading to a better user experience.

Keep up the good work, and don't hesitate to ask if you have any questions. Happy coding! 🌟💻🚀